styles.css

```diff
@@ -1,11 +1,119 @@
-*, *:before, *:after {
+*,
+*:before,
+*:after {
     box-sizing: border-box;
 }
 
-body, html {
+body,
+html,
+a {
     font-family: 'Poppins', sans-serif;
     margin: 0;
     padding: 0;
     color: #293241;
     font-size: 18px;
+    text-decoration: none;
+}
+
+.page-header {
+    height: 82px;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.nav-bar {
+    display: flex;
+    justify-content: space-between;
+    gap: 64px;
+    margin: 18px 100px;
+}
+
+.meniu {
+    display: flex;
+    align-items: center;
+}
+
+.frames ol {
+    display: flex;
+    flex-direction: row;
+    margin-top: 16px;
+    margin-right: 81px;
+    padding: 0px;
+    gap: 64px;
+    list-style-type: none;
+}
+
+a {
+    color: #293241;
+}
+
+a:hover {
+    font-weight: 700;
+    cursor: pointer;
+}
+
+.button:hover {
+    background-color: #8b5d5d;
+    border: 1px solid #000000;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    cursor: pointer;
+}
+
+.button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 43px;
+    width: 150px;
+    background-color: #EE6C4D;
+    border-radius: 48px;
+    border: none;
+    font-weight: 700;
+    font-size: 18px;
+    color: #FFFFFF;
+}
+
+.section {
+    display: flex;
+    justify-content: center;
+    height: 812px;
+}
+
+.presentation {
+    display: flex;
+    flex-direction: column;
+    margin: 100px 20px;
+    width: 1233.98px;
+    height: 631.05px;
+    background-image: url(images/hero.png);
+    background-position: right 60%;
+    background-repeat: no-repeat;
+}
+
+.title {
+    margin-top: 160px;
+    width: 494px;
+    height: 225px;
+    color: #EE6C4D;
+    font-size: 50px;
+    font-weight: 700;
+}
+
+.title p,
+.description p {
+    margin-top: 0px;
+}
+
+.description {
+    display: flex;
+    width: 360px;
+    height: 90px;
```

Comment:
aukščio nebereikia rašyti, nes yra `line-height`. Pagal linijos aukštį automatiškai gaunasi 90, nes trys eilutės po 30. Jeigu pvz esant responsive dizainui galėtų keistis plotis ir pasidarytų dvi arba keturios eilutės dėl to įrašyto aukščio išsikraipytų dizainas.